### ISSUE TITLE
Connecting to battle.net creates a new account.

### ISSUE THREAD

**Issue by adriankarlen** [oratory/wago.io#45]
Don't know if this is intended behaviour or a bug. But I decided to connect my battle.net-account and I now have two wago accounts. Shouldn't it just connect the existing account to a bnet-id?

**Comment by oratory** [comment]
If you're logged into Wago already and then connect battlenet then yes it will add your battlenet account to your Wago account. If you're not logged in then it will create a new account.

One thing I do need is a way to merge or delete accounts since your issue does happen now and then.

**Comment by guicaulada** [comment]
Yup, same issue here, tried to connect my Battle.net account, somehow endedup on a new account, now trying to repeat the process just logs me in to the other account instead of merging them.

**Comment by TeDSlash** [comment]
I will ask here, because its almost the same problem. 

I am logged into my wago account and want to connect to my battle.net one. But after forwarding i can just click "Log in". 
What Username is he using? I can't change it somewhere. It's strange, because it says "We can't log in with these informations." So what is he using?

Can someone help?
